pkg/metrics/certificates.go

```diff
@@ -46,6 +47,7 @@ func (m *Metrics) UpdateCertificate(ctx context.Context, crt *cmapi.Certificate)
 
 	m.updateCertificateStatus(key, crt)
 	m.updateCertificateExpiry(ctx, key, crt)
+	m.updateCertificateRenewBefore(crt)
```

Comment:
It looks like the comment at line 38 above should be updated as well.

Reply:
Thank, missed that.